src/main/java/org/motechproject/mots/web/ModuleAssignmentController.java

```diff
@@ -67,4 +67,19 @@ public ChwModulesDto getAssignedModules(@RequestParam(value = "chwId") String ch
 
     return moduleAssignmentMapper.toDto(assignedModules);
   }
+
+  /**
+   * Get Modules with started CHW ModuleProgress assigned to CHW.
+   * @param chwId id of CHW
+   * @return modules assigned to CHW
+   */
+  @RequestMapping(value = "/startedAssignedModules", method = RequestMethod.GET)
+  @ResponseStatus(HttpStatus.OK)
+  @ResponseBody
+  public ChwModulesDto getStartedAssignedModules(@RequestParam(value = "chwId") String chwId) {
```

Comment:
remove this, later to recognize the not started modules on the UI, the ChwModulesDto will be changed to store this information